<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupWidget</class>
 <widget class="QDialog" name="SetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StageRunner Setup &amp; Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="dmxTab">
      <attribute name="title">
       <string>DMX</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>QLC plugins</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="handleWidth">
          <number>8</number>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="handleWidth">
           <number>8</number>
          </property>
          <widget class="QListWidget" name="qlcPluginsList"/>
          <widget class="QTableWidget" name="dmxMappingTable"/>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTextEdit" name="qlcPluginEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="configurePluginButton">
             <property name="text">
              <string>Configure plugin</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="audioTab">
      <attribute name="title">
       <string>Audio</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Audio buffer size (bytes)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="audioBufferSizeEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="prohibitAudioDoubleStartCheck">
         <property name="text">
          <string>Prohibit double start</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="reactivateAudioLabel">
         <property name="text">
          <string>reactivate after (ms)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="reactivateAudioTimeSpin">
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="singleStep">
          <number>25</number>
         </property>
         <property name="value">
          <number>800</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="mapAudioToChannelLabel1">
         <property name="text">
          <string>Map Channel 1 to DMX</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="mapAudioToDmx1">
         <property name="maximum">
          <number>512</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="mapAudioToUniv1">
         <property name="maximum">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="mapAudioToChannelLabel1_2">
         <property name="text">
          <string>Map Channel 2 to DMX</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="mapAudioToUniv2">
         <property name="maximum">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSpinBox" name="mapAudioToDmx2">
         <property name="maximum">
          <number>512</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="mapAudioToChannelLabel1_3">
         <property name="text">
          <string>Map Channel 3 to DMX</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="mapAudioToUniv3">
         <property name="maximum">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QSpinBox" name="mapAudioToDmx3">
         <property name="maximum">
          <number>512</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="mapAudioToChannelLabel1_4">
         <property name="text">
          <string>Map Channel 4 to DMX</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="mapAudioToUniv4">
         <property name="maximum">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="2">
        <widget class="QSpinBox" name="mapAudioToDmx4">
         <property name="maximum">
          <number>512</number>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="playlistChannelSpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="playlistChannelLabel">
         <property name="text">
          <string>FX Audio playlist default channel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="guiTab">
      <attribute name="title">
       <string>GUI</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Interface Style</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="appStyleLabel">
            <property name="text">
             <string>Application style</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="appStyleCombo"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="dialKnobStyleLabel">
            <property name="text">
             <string>Dial knob style</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="dialKnobStyleCombo"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="noInterfaceFeedbackCheck">
         <property name="text">
          <string>Enable DMX channel feedback for non existing interfaces</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="showVolumeDialGroup">
         <property name="title">
          <string>Show Volume Dial</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="enableFFTGroup">
         <property name="title">
          <string>Enable FFT</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="saveDmxConfigButton">
       <property name="text">
        <string>Save DMX Config</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>prohibitAudioDoubleStartCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>reactivateAudioLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
